partner_email_check/README.rst

```diff
@@ -3,21 +3,20 @@
    :alt: License: AGPL-3
 
 ===========
-Email check
+Partner Email Check
 ===========
```

Comment:
These lines must match the length of the title. Otherwise you get rendering warnings.